src/redux/missions/missions.js

```diff
@@ -0,0 +1,36 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const initialState = { missions: [] };
+
+const URL = 'https://api.spacexdata.com/v3/missions';
+
+export const fetchMissions = createAsyncThunk('GET/fetchMissions/fulfilled', async () => {
+  const response = await fetch(URL);
+  const data = await response.json();
+  return data;
+});
+
+const missionSlice = createSlice({
+  name: 'GET',
+  initialState,
+  reducers: {
+    toogleReserve: (state, { payload }) => {
+      state.missions = state.missions.map((mission) => { // eslint-disable-line no-param-reassign
+        if (mission.mission_id === payload) {
+          return { ...mission, reserved: !mission.reserved };
+        } return mission;
+      });
+    },
+  }, /* eslint-disable no-param-reassign */
+  extraReducers: (builder) => {
+    builder
+      .addCase(fetchMissions.fulfilled, (state, { payload }) => {
+        state.missions = payload.map((x) => ({ ...x, reserved: false }));
+      });
+  },
+  /* eslint-enable no-param-reassign */
+});
```

Comment:
- [ ] Let's try to not disable Linter checks in our program as they are there to help us write better and more professional code. Disabling Linter checks defies its purpose and doesn't help you become a professional developer. Learning how to fix them now will set you up ready for the industry as that's the standard. Please fix them👍🏽